docs/intro/install.rst

```diff
@@ -102,10 +102,8 @@ just like any other Python package.
 (See :ref:`platform-specific guides <intro-install-platform-notes>`
 below for non-Python dependencies that you may need to install beforehand).
 
-Python virtualenvs can be created to use Python 2 by default, or Python 3 by default.
-
-* If you want to install scrapy with Python 3, install scrapy within a Python 3 virtualenv.
-* And if you want to install scrapy with Python 2, install scrapy within a Python 2 virtualenv.
+Python virtualenvs can be created to use Python 2 by default, or Python 3 by default. As Scrapy
+only supports Python 3, make sure you created a Python 3 virtualenv.
```

Comment:
I was going to suggest updating the documentation links to point to https://docs.python.org/3/library/venv.html. However, I would like to propose a complete rewrite of this section once these changes are merged (#4001), so no need to worry too much about this part in this pull request.